src/v3/test/integration/__snapshots__/authenticator-enroll-select-authenticator-with-skip.test.tsx.snap

```diff
@@ -115,7 +116,7 @@ exports[`authenticator-enroll-select-authenticator renders form 1`] = `
                     <div
                       class="ods-4o5zYQ ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs title"
                     >
-                      Okta Password
```

Comment:
because of properties file update?

Reply:
Yes, `Okta Password` was never valid, it was using the label that was sent from the ion response, it is now using the proper i18n key.